.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,7 +21,6 @@ Fixes #
 **Slash Commands**
 
 You can write slash commands (`/command`) in the first line of a comment to perform
-specific operations. Supported slash commands are:
+specific operations. Supported slash command is:
 
 - `/format`: automatically format and lint the code
-- `/test-gmt-dev`: run full tests on the latest GMT development version
```

Comment:
Similar to my comment at https://github.com/GenericMappingTools/pygmt/pull/2958#discussion_r1443947533, should we still document this somewhere so that contributors/maintainers know to request that the `run/test-gmt-dev` label gets added to PRs that require it? Or just rely on maintainers to know when to set the label?

Reply:
See https://github.com/GenericMappingTools/pygmt/pull/2958/files#r1443950923.